templates/bot/index.js

```diff
@@ -1,10 +0,0 @@
-"use strict";
```

Comment:
I didn't see the generic bot template replace anywhere.

Reply:
Not sure I follow. Are you saying you want to keep this example and set it as `generic` for the "type" option of the command?